pkg/kubehound/graph/builder.go

```diff
@@ -71,7 +71,7 @@ func (b *Builder) buildEdge(ctx context.Context, label string, e edge.Builder, o
 		return err
 	}
 
-	err = e.Stream(ctx, b.storedb, b.cache,
+	err = e.Stream(ctx, b.storedb, b.cache, &b.cfg.Dynamic,
```

Comment:
Question (applies to all the caller as well):
Shouldn't this config be set in the `Initialize(cfg)` method instead (line 65)?

This would enable us to not modify the signature of this function with a specific config now (I believe this would make extending this with new configuration easier as well.)

All the caller would only have to change from `runtime.ClusterName` to `e.runtime.ClusterName` for example if you call that config field "runtime" as well.

(and make the diff a bit smaller as well haha)

Reply:
Yes good point. Did not see the inititate method.